src/library.cpp

```diff
@@ -121,6 +121,9 @@ void Library::save()
 {
     m_library.writeToFile(appendToDirectory(getDataDirectory(),"library.xml"));
     m_library.writeBookmarksToFile(appendToDirectory(getDataDirectory(), "library.bookmarks.xml"));
+    auto manipulator = LibraryManipulator(this);
+    auto manager = kiwix::Manager(&manipulator);
+    manager.readFile(appendToDirectory(getDataDirectory(),"library.xml"), false);
```

Comment:
This seems unnecessary. Loading something we just save should "return" the same thing.
If not, it is a bug in the kiwix-lib and it should be fixed there.

I think the best here is to simply remove this commit and open an issue to keep a trace of this change and investigate further.